jumpscale/core/config/config.py

```diff
@@ -115,6 +117,28 @@ def update_config(data):
         toml.dump(data, f)
 
 
+def get(key):
+    """ Retrives value from jumpscale config
+
+    Arguments:
+        key {str} -- the key you wish to retrieve
```

Comment:
We use the following style when declaring arguments (which works well with doc generator):

```
Arg...
    key (str): the key to ...
```

Reply:
updated